physical_infra_provider_rest_api/usage/topics/get_physical_servers.adoc

```diff
@@ -0,0 +1,27 @@
+=== Get Physical Servers
+
+. Log in to CloudForms Management Engine API as a user who has permission to access the API. The default user is admin, password smartvm.
+
+. Use the following url structure for access the api **"<CLOUDFORMS ADDRESS>:<PORT>/api/" (ex. localhost:3000/api/)**.
+
+. Use the following request for get all physical servers. **(ex. localhost:3000/api/physical_servers)**.
+
+. It’s returned a JSON with the following keys**
```

Comment:
Maybe you can put the keys into a table like you did in step 4 of section 1.5 (Get a Physical Server by ID).